spec/spec.tex

```diff
@@ -829,6 +829,15 @@ \subsubsection{Read Under Write Behaviour}
 
 If the read-under-write flag is set to \verb|undefined|, then the value held by the read port after the appropriate read latency is undefined.
 
+
+For the purpose of defining such collisions, an ``active write port'' is either a write port or readwrite port that is used to initiate a write operation on a given clock edge.
+An ``active read port'' is a read port or readwrite port that is used to initiate a read operation on a given clock edge.
```

Comment:
Maybe add that this means that `en=1 and wmode=1`.

Reply:
That's a good idea.